projects/git-scm.org/package.yml

```diff
@@ -47,6 +47,13 @@ test: |
   git add .
   test "$(git diff --name-only --cached)" = "testfile"
 
+  # necessary so `git commit` will work
+  git config --global user.email "you@example.com"
+  git config --global user.name "Your Name"
```

Comment:
probably doesn't need to be `--global`, right?

Reply:
oh this effects the self-hosted runners indeed.